### DISCUSSION TITLE
Does SAPIEN provide suction gripper?

### DISCUSSION THREAD

**Discussion by keiohta** [haosulab/SAPIEN#67] [Q&A]
Hi author, thanks again for continuous supports and developing a great simulator.

I would like to use a suction glipper in our simulation, just like this repo of [ravens](https://github.com/google-research/ravens) (also known as Transporter Network), but not sure how to do that. Can you tell me whether SAPIEN provides such functionality?

**Comment by fbxiang** [comment]
Does this issue answer your question? I think it has the same use case as your case. https://github.com/haosulab/SAPIEN/issues/49

**Reply by keiohta** [reply]
Thanks for the quick reply. Let me check the issue and I'll post again if it won't solve my problem or I have other questions.
